.github/workflows/build_maven.yml

```diff
@@ -33,5 +33,5 @@ jobs:
     with:
       envs: >-
         {
-          "SKIP_SPARK_RELEASE_VERSIONS": "3.3.4,3.4.2,3.5.0"
+          "SKIP_SPARK_RELEASE_VERSIONS": "3.4.2,3.5.0,3.5.1"
```

Comment:
`3.3.4` is removed and `3.5.1` is added. For now, 3.5.0 is not removed yet.

https://dist.apache.org/repos/dist/release/spark/

![Screenshot 2024-02-21 at 21 54 13](https://github.com/apache/spark/assets/9700541/acb3607c-d303-4964-94bb-adc1d4dcf770)